packages/components/src/border-box-control/border-box-control/hook.ts

```diff
@@ -39,6 +40,12 @@ export function useBorderBoxControl(
 		...otherProps
 	} = useContextSystem( props, 'BorderBoxControl' );
 
+	maybeWarnDeprecated36pxSize( {
```

Comment:
Can we use this new utility for any existing component?

Reply:
Everything except for `FormFileUpload` and `Button`, which will need more specific conditional logic.